packages/mobile/src/constants/storage-keys.ts

```diff
@@ -0,0 +1,7 @@
+export const DEVICE_TOKEN = '@device-token'
```

Comment:
Love the consistency here 😂

Reply:
It's sad too cause we can't really change them, lol